From 5f907445551685bd30af7567c08662f7100a0d70 Mon Sep 17 00:00:00 2001
From: Andrei Vagin <avagin@gmail.com>
Date: Wed, 22 Dec 2021 17:06:21 -0800
Subject: [PATCH 093/245] test: log testname.out.inprogress if a test has
 failed

This is required if the test failed by timeout.

Signed-off-by: Andrei Vagin <avagin@gmail.com>
---
 test/zdtm.py | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/test/zdtm.py b/test/zdtm.py
index 086ae0285..c0e24a41a 100755
--- a/test/zdtm.py
+++ b/test/zdtm.py
@@ -589,11 +589,12 @@ class zdtm_test:
             os.unlink(self.__pidfile())
 
     def print_output(self):
-        if os.access(self.__name + '.out', os.R_OK):
-            print("Test output: " + "=" * 32)
-            with open(self.__name + '.out') as output:
-                print(output.read())
-            print(" <<< " + "=" * 32)
+        for postfix in ['.out', '.out.inprogress']:
+            if os.access(self.__name + postfix, os.R_OK):
+                print("Test output: " + "=" * 32)
+                with open(self.__name + postfix) as output:
+                    print(output.read())
+                print(" <<< " + "=" * 32)
 
     def static(self):
         return self.__name.split('/')[1] == 'static'
-- 
2.35.1

